src/utils.ts

```diff
@@ -800,18 +801,30 @@ export class Utils {
         return '\n\n # \n\n The above Job Summary was generated by the <a href="https://github.com/marketplace/actions/setup-jfrog-cli"> Setup JFrog CLI GitHub Action </a>';
     }
 
+    /**
+     * Checks if the header image is accessible via the internet.
+     * saves the result in a static variable to avoid multiple checks.
+     * @private
+     */
     private static async isHeaderPngAccessible(): Promise<boolean> {
+        if (this.isSummaryHeaderAccessible != undefined) {
+            return this.isSummaryHeaderAccessible;
+        }
         const url: string = this.MARKDOWN_HEADER_PNG_URL;
         const httpClient: HttpClient = new HttpClient();
         try {
-            const response: HttpClientResponse = await httpClient.head(url);
-            return response.message.statusCode === 200;
+            const requestOptions: OutgoingHttpHeaders = {
+                socketTimeout: 5000, // Set timeout to 5 seconds
```

Comment:
Let's place the comment above the line it describes, and not to the side of it, according to our convention.